src/content-cli-config.ts

```diff
@@ -22,6 +22,21 @@ export class Config {
 
         return program;
     }
+
+    public static export(program: CommanderStatic): CommanderStatic {
+        program
+            .command("export")
+            .description("Command to export package configs")
```

Comment:
Do we want to mention packages here?

Edit: We can think more on this before releasing.

Reply:
I will note this topic for discussion before release and mark this comment as resolved.